ios/brave-ios/Sources/Brave/Frontend/UserContent/UserScripts/AllFrames/AtDocumentEnd/ContextMenu.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+"use strict";
+
+// Ensure this module only gets included once. This is
+// required for user scripts injected into all frames.
+window.__firefox__.includeOnce("ContextMenu", function($) {
+  window.addEventListener("touchstart", function(evt) {
```

Comment:
Basically every time a user scrolls the page it will send `postMessage` request to native. Doesn't seems great from the performance perspective. Could we detect a long press somehow instead?